setup.py

```diff
@@ -89,8 +89,9 @@
       install_requires=['posttroll>=1.3.0',
                         'trollsift',
                         'pyyaml'],
-      tests_require=['trollsift', 'netifaces', 'watchdog', 'posttroll', 'pyyaml', 'pyinotify', 's3fs',
+      tests_require=['trollsift', 'netifaces', 'watchdog', 'posttroll', 'pyyaml',
+                     'pyinotify', 's3fs', 'freezegun',
                      'pyresample', 'python-dateutil', 'posttroll', 'pytest'],
       extras_require=extras_require,
-      python_requires='>=3.7',
+      python_requires='>=3.8',
```

Comment:
The changes for CI is dropping 3.8, so should this also have 3.9?

Reply:
yes